packages/zowe-explorer/src/extension.ts

```diff
@@ -161,66 +175,84 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
         initJobsProvider(context, jobsProvider);
     }
     if (datasetProvider || ussFileProvider) {
-        vscode.commands.registerCommand("zowe.openRecentMember", () =>
-            sharedActions.openRecentMemberPrompt(datasetProvider, ussFileProvider)
+        context.subscriptions.push(
```

Comment:
From what I'm gathering, all registered commands are now part of the context subscriptions.
That way the commands are properly de-registered by VSCode when the extension is removed/disabled
Is that the intention here as well ?

Source: https://stackoverflow.com/a/55579618/16555983